taipy/core/job/job.py

```diff
@@ -42,8 +42,16 @@ def __run_callbacks(job):
 class Job(_Entity, _Labeled):
     """Execution of a `Task^`.
 
-    A job handles the status of the execution, contains the stacktrace of exceptions that were
-    raised during the execution, and notifies subscribers on status change.
+    Tasks, Sequences, and Scenarios entities can be submitted for execution. The submission
+    of a scenario triggers the submission of all the contained tasks. Similarly, the submission
+    of a sequence also triggers the execution of all the ordered tasks.
+
+    Every time a task is submitted for execution, a new _Job_ is created. A job represents a
```

Comment:
I believe we want to use * for italic text

```suggestion
    Every time a task is submitted for execution, a new *Job* is created. A job represents a
```